scripts/parser-tests/typescript/allowlist.txt

```diff
@@ -23,11 +23,8 @@ classExpressionWithDecorator1.ts
 classExtendsMultipleBaseClasses.ts
 classOverloadForFunction.ts
 classWithEmptyTypeParameter.ts
-# TypeScript doesn't allow a parameter to be named arguments even in non-strict mode, which we don't catch.
```

Comment:
Can we modify the generator to preserve such comments?

Reply:
They were already supported but they must be inline